### ISSUE TITLE
Does it work?

### ISSUE THREAD

**Issue by joelreymont** [Shinmera/cl-graphql#1]
Would love to use it on top of AllegroCache!

**Comment by Shinmera** [comment]
No. I could never figure out a good way to map this mess of a protocol to a sensible way to interact with it. Currently it's too low on my priority list to give it the (probably like tenth) repeated try.
